tests/test_utils.py

```diff
@@ -64,4 +64,62 @@ def test_run_command(caplog):
         and_return(subprocess.CompletedProcess(cmd, 0, "stdout", "stderr"))
     res = run_command(cmd, log_level=log_level, additional_env=additional_env, combine_out_err=combine_out_err)
     assert secret not in caplog.text
-    assert REDACTED_OUT_SECRET in caplog.text
\ No newline at end of file
+    assert REDACTED_OUT_SECRET in caplog.text
+
+
+def test_change_cwd():
+    present_dir = os.getcwd()
+    with tempfile.TemporaryDirectory() as target_dir:
+        with change_cwd(target_dir):
+            assert os.getcwd() == os.path.realpath(target_dir)
+        assert os.getcwd() == present_dir
+
+
+def test_get_current_commit(caplog):
+    current_commit = subprocess.run(["git", "rev-parse", "--short", "HEAD"],text=True, capture_output=True)
```

Comment:
This feels like something we should mock out instead of assuming the current working directory is a `git` directory. 

Can we do something like what flexmock is doing above - https://github.com/DataDog/apigentools/pull/71/files#diff-87010571bbeeee0b9a13279c493670f3R42

The exception case is good, since we know the new tmp directory won't be a git folder 😄